src/main/java/net/sf/jabref/gui/RemoteConnectionLostDialog.java

```diff
@@ -0,0 +1,48 @@
+package net.sf.jabref.gui;
+
+import javax.swing.JOptionPane;
+
+import net.sf.jabref.event.RemoteConnectionLostEvent;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.model.database.DatabaseLocation;
+
+import com.google.common.eventbus.Subscribe;
+
+public class RemoteConnectionLostDialog {
+
+    private final JabRefFrame jabRefFrame;
+
+
+    public RemoteConnectionLostDialog(JabRefFrame jabRefFrame) {
+        this.jabRefFrame = jabRefFrame;
+    }
+
+    @Subscribe
+    public void listen(RemoteConnectionLostEvent connectionLostEvent) {
+
+        jabRefFrame.output(Localization.lang("Connection lost."));
+
+        String[] options = {Localization.lang("Reconnect"), Localization.lang("Work offline"),
+                Localization.lang("Close database")};
+
+        int answer = JOptionPane.showOptionDialog(jabRefFrame,
+                Localization.lang("The connection to the server has been determinated.") + "\n\n",
+                Localization.lang("Connection lost"), JOptionPane.YES_NO_CANCEL_OPTION, JOptionPane.WARNING_MESSAGE,
+                null, options,
+                options[0]);
+
+        if (answer == 0) {
+            jabRefFrame.closeCurrentTab();
+            OpenRemoteDatabaseDialog openRemoteDBDialog = new OpenRemoteDatabaseDialog(jabRefFrame);
+            openRemoteDBDialog.setLocationRelativeTo(jabRefFrame);
```

Comment:
Please move this line to the OpenDatabaseDialog class so you can remove this line here and the one in `OpenRemoteDatabaseAction`